contrib/src/main/java/com/twelvemonkeys/contrib/tiff/TIFFUtilities.java

```diff
@@ -517,6 +664,7 @@ public void rotate(int degree) {
     /**
      * TODO: Temporary clone, to be removed after TMI204 has been closed
      */
+
```

Comment:
The TIFFBaseline / TIFFExtension / TIFFCustom are still only package visible andlocated at the TIFF plugin. We explicitly removed the dependencies to the TIFF plugin when I implemented the utilities, so the reason for the copy still exists.
I propose to handle that like com.twelvemonkeys.imageio.metadata.exif.TIFF and com.twelvemonkeys.imageio.metadata.tiff.TIFF: Move those classes to the metadata plugin and keep a deprecated interface at the old location, so not everything must be changed at once.

I propose to handle that like com.twelvemonkeys.imageio.metadata.exif.TIFF and com.twelvemonkeys.imageio.metadata.tiff.TIFF: Move those classes and keep an deprecated interface at the old location, so not everything must be changed at once.